Client/BluetoothWrapper.cpp

```diff
@@ -66,11 +90,25 @@ void BluetoothWrapper::_waitForAck()
 
 	do
 	{
-		auto numRecvd = this->_connector->recv(buf, sizeof(buf));
```

Comment:
the whole change here is unclear - now two different pieces of logic are parsing the messages. Delete one of them and use the constants for both END and START markers.